Automattic/wp-calypso
# Launchpad: Adds a body to tasks

Related to #

## Proposed Changes

* For my 20% time, I decided to test an idea we discussed in one of our calls with @daledupreez & @nuriapenya. The idea is to have a body with a description of the task and some other components like buttons, links, images, videos, etc.
* To enable this functionality, I also worked on allowing the backend to manage the content within the task body. You can find the details and progress in this pull request: https://github.com/Automattic/jetpack/pull/33489.

<img width="327" alt="Screen Shot 2023-10-19 at 15 50 56" src="https://github.com/Automattic/wp-calypso/assets/1234758/2adf1ed7-18f5-4210-92e2-f6dfb721cc6f">

## Testing Instructions



* Follow the instructions on https://github.com/Automattic/jetpack/pull/33489 to set up your sandbox
* Navigate to `/home/:siteSlug/flags=launchpad/navigator` on a site with the build intent.
* Click on the Launchpad Navigator icon
* You should see the "Customize your domain" task with a description and two buttons.

## Pre-merge Checklist
